tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -0,0 +1,61 @@
+package cmd
```

Comment:
This is really the net-new code (aside from all the code organization as `cobra` commands, of course) that deals with generating the package metadata files.